core/testing/build.gradle.kts

```diff
@@ -15,6 +15,7 @@
  */
 plugins {
     id("nowinandroid.android.library")
+    id("nowinandroid.android.library.compose")
```

Comment:
Why does the testing module need the compose plugin?

Reply:
Ah, because it has Compose dependencies which use the version provided by the BOM 👍